spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
           attribute DOMString? placeholder;
-          attribute double? rangeValue;
-          attribute double? rangeMin;
-          attribute double? rangeMax;
-          attribute double? rangeStep;
+          attribute double? valueNow;
+          attribute double? valueMin;
+          attribute double? valueMax;
 
           // Live regions.
-          attribute boolean? liveAtomic;
-          attribute boolean? liveBusy;
-          attribute LiveType? liveStatus;
-          attribute LiveRelevantType? liveRelevant;
+          attribute boolean? atomic;
+          attribute boolean? busy;
+          attribute LiveType? live;
+          attribute LiveRelevantType? relevant;
 
-          // Relationships.
+          // Other relationships.
           attribute AccessibleNode? activeDescendant;
           attribute iterable&lt;AccessibleNode&gt; controls;
-          attribute iterable&lt;AccessibleNode&gt; describedBy;
           attribute AccessibleNode? details;
           attribute AccessibleNode? errorMessage;
           attribute AccessibleNode? flowTo;
-          attribute iterable&lt;AccessibleNode&gt; labeledBy;
+          attribute iterable&lt;AccessibleNode&gt; owns;
```

Comment:
From F2F: Possibly no need to include `owns` b/c this would ultimately be replaced by the children array, or a standard DOM operation would be sufficient, no?